src/Bridge/Symfony/Bundle/ErrorHandler/SymfonyExceptionHandler.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+/*
+ * @author     mfris
+ * @copyright  PIXELFEDERATION s.r.o.
+ * @license    Internal use only
```

Comment:
well obviously I cannot accept code with licenses/copyright please remove it

Reply:
done